lua/damagelogs/client/tabs/rdm_manager.lua

```diff
@@ -320,13 +326,17 @@ local function TakeAction()
         slaynr2:AddOption(TTTLogTranslate(GetDMGLogLang, "ReportedPlayer") .. " (" .. report.attacker_nick .. ")", function()
             if IsValid(attacker) then
                 if ulx then
-                    RunConsoleCommand("ulx", mode == 1 and "aslayid" or "ajailid", report.attacker, "0")
-                else
-                    serverguard.command.Run("raslay", false, attacker:Nick())
+                    RunConsoleCommand("ulx", mode == 1 and "aslay" or "ajail", attacker:Nick(), "0")
```

Comment:
Can this go back to `aslayid`/`ajailid`?

I remember this command having problems in the past with players that had weird characters in their names.

Reply:
done